modules/reference/pages/rpk/rpk-cloud/rpk-cloud-byoc.adoc

```diff
@@ -2,14 +2,13 @@
 
 Manage a Redpanda cloud BYOC agent.
 
-For BYOC, Redpanda installs an agent service in your owned cluster. The agent
-then proceeds to provision further infrastructure and eventually, a full
-Redpanda cluster.
-
-The BYOC command runs Terraform to create and start the agent. You first need
-a redpanda-id (or cluster ID); this is used to get the details of how your
-agent should be provisioned. You can create a BYOC cluster in our cloud UI
-and then come back to this command to complete the process.
+include::reference:partial$rpk-cloud-byoc-concept.adoc[]
+
+For example, you can create a BYOC cluster in AWS by executing this command: 
```

Comment:
```suggestion
For example, to create a BYOC cluster in AWS, run: 
```

Reply:
removed example. Will be added a new a ticket